test/test_nn_quantized.py

```diff
@@ -47,6 +47,43 @@ def test_linear_api(self):
         Z_ref = torch.ops.quantized.fbgemm_linear(X_q, W_pack, B_q, out_scale, out_zero_point)
         self.assertEqual(Z_ref, Z_q)
 
+        # Test serialization of quantized Linear Module using state_dict
+        model_dict = qLinear.state_dict()
+        self.assertEqual(model_dict['weight'], W_q)
+        with tempfile.NamedTemporaryFile() as f:
+            torch.save(model_dict, f)
+            f.seek(0)
+            loaded_dict = torch.load(f)
+        for key in model_dict:
+            self.assertEqual(model_dict[key], loaded_dict[key])
+        # TODO: should we do this?
+        qLinear2 = nnq.Linear.from_float(torch.nn.Linear(output_channels, input_channels))
```

Comment:
@dzhulgakov, here we need to construct a dummy nnq.Linear module, how should we do that?